releases/_posts/2023-04-13-spark-release-3-4-0.md

```diff
@@ -0,0 +1,420 @@
+* Support timestamp in seconds for TimeTravel using Dataframe options ([SPARK-39633](https://issues.apache.org/jira/browse/SPARK-39633))
+* Add SparkSession.config(Map) ([SPARK-40163](https://issues.apache.org/jira/browse/SPARK-40163))
+* Support changing session catalog's default database ([SPARK-35242](https://issues.apache.org/jira/browse/SPARK-35242))
+* Protobuf support for Spark - from_protobuf AND to_protobuf ([SPARK-40654](https://issues.apache.org/jira/browse/SPARK-40654))
+* Add WHEN NOT MATCHED BY SOURCE clause to MERGE INTO ([SPARK-40921](https://issues.apache.org/jira/browse/SPARK-40921))
+* Relax ordering constraint for CREATE TABLE column options ([SPARK-40944](https://issues.apache.org/jira/browse/SPARK-40944))
+* SQL Equivalent for Dataframe overwrite command ([SPARK-40956](https://issues.apache.org/jira/browse/SPARK-40956))
+* Support Generate with no required child output to host outer references ([SPARK-41441](https://issues.apache.org/jira/browse/SPARK-41441))
+* ORDER BY ALL ([SPARK-41637](https://issues.apache.org/jira/browse/SPARK-41637))
+* GROUP BY ALL ([SPARK-41635](https://issues.apache.org/jira/browse/SPARK-41635))
+* Add flatMapSortedGroups and cogroupSorted ([SPARK-38591](https://issues.apache.org/jira/browse/SPARK-38591))
+* Support subqueries with correlated non-equality predicates ([SPARK-36114](https://issues.apache.org/jira/browse/SPARK-36114))
+* Support subqueries with correlation through UNION/INTERSECT/EXCEPT ([SPARK-36124](https://issues.apache.org/jira/browse/SPARK-36124))
+
+#### ANSI
+
+* ANSI mode: always return null on invalid access to map column ([SPARK-40066](https://issues.apache.org/jira/browse/SPARK-40066))
+* Support double quoted identifiers ([SPARK-40585](https://issues.apache.org/jira/browse/SPARK-40585))
+* ANSI SQL mode: Round/Bround should return an error on integer overflow [SPARK-42045](https://issues.apache.org/jira/browse/SPARK-42045))
+* Support casting of integrals to ANSI intervals ([SPARK-40008](https://issues.apache.org/jira/browse/SPARK-40008))
+* Support cast of decimals to ANSI intervals ([SPARK-40014](https://issues.apache.org/jira/browse/SPARK-40014))
+* Return wider ANSI interval types from the percentile functions ([SPARK-40151](https://issues.apache.org/jira/browse/SPARK-40151))
+* Support cast of ANSI intervals to decimals ([SPARK-39470](https://issues.apache.org/jira/browse/SPARK-39470))
+* Support casting intervals to integrals in ANSI mode ([SPARK-39451](https://issues.apache.org/jira/browse/SPARK-39451))
+* Harden SQLSTATE usage for error classes ([SPARK-41994](https://issues.apache.org/jira/browse/SPARK-41994))
+
+#### Functions
+
+* Support table-valued generator functions in the FROM clause ([SPARK-41594](https://issues.apache.org/jira/browse/SPARK-41594))
+* Support ANSI Aggregate Function: REGR_SXY ([SPARK-37681](https://issues.apache.org/jira/browse/SPARK-37681))
```

Comment:
Shall we group all the `ANSI Aggregate Function` together?

Reply:
Updated.